packages/prosemirror-lwdita/src/commands.ts

```diff
@@ -244,16 +245,16 @@ export function renderPrDialog(ghrepo: string, source: string, branch: string, u
 
       // Create a PR from the contribution
       createPrFromContribution(ghrepo, source, branch, updatedXdita, title, description)
-      .then((prURL: string) => {
-        console.log('The document has been published to GitHub.');
-        console.log('pr url:', prURL);
-        // Show a user notification with the successful result and a link to the PR
-        showPublicationResultSuccess(prURL)
-      }).catch((error: Error) => {
-        console.error('Error:', error);
-        // show error dialog
-        showPublicationResultError(error.message)
-      });
+        .then((prURL: string) => {
```

Comment:
Why is there changes in here? I don't see anything changed in the code, is it just whitespace?

Reply:
@marmoure Yes, seems to be a whitespace change.